src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/DRConfig/SetAzureEventHubDRConfigurationBreakPairing.cs

```diff
@@ -0,0 +1,58 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.EventHub.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.EventHub.Commands.GeoDR
+{
+    /// <summary>
+    /// 'New-AzureRmEventHubDRConfigurationsBreakPairing' Cmdlet disables the Disaster Recovery and stops replicating changes from primary to secondary namespaces
```

Comment:
@v-Ajnava typo: the cmdlet mentioned is `New` rather than `Set`

Reply:
Resolved